index.html

```diff
@@ -0,0 +1,152 @@
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <link rel="stylesheet" type="text/css" href="assets/css/style.css">
+
+    <link rel="resetsheet" type="text/reset" href="assets/css/reset.css">
+
+    <title>Game day</title>
+</head>
+<body>
+<div class="container">
+
+    <div class="row">
+
+        <div id="header">
+            <h1>   Crystal Collector!</h1>
+        </div>
+
+    </div>
+
+    <div class="row">
+
+        <div id="instructions">
+            <p> You will be give a random number at the start of the game.   </p>  
+            <p>There are four crystals below. By clicking on a crystal you will add a specific amount of points to your total score.</p>    
+            <p>You win the game by matching your total score to random number, you lose the game if your total score goes above the random number.</p>
+            <p>The value of each crystal is hidden from you until you click on it.</p>
+            <p>Each time when the game starts, the game will change the values of each crystal.</p>
+        </div>
+        
+    </div>
+
+    <div class="row">
+
+        <div class="col-sm-3">
+            <div id="scorecard1">
+                <h3 id=compScore> </h3>
+            </div>    
+        </div>
+
+        <div class="col-sm-3">
+            <div id="scorecard2">
+            <p id=win/loss>  </p> <br>
+            <p id=wins> Wins: </p> <br>
+            <p id=losses> Losses: </p> <br>
+            </div> 
+        </div>
+
+        <div class="col-sm-6">
+            <div>
+
+            </div>
+            
+        </div>
+          
+    </div>
+
+    <div class="row">
+
+        <div id="images">
+            <img class="crystal-images" id="crystal1" src="assets/images/Boulder_Badge.png" alt="crystal-1">
+            <img class="crystal-images" id="crystal2" src="assets/images/Rainbow_Badge.png" alt="crystal-2">
+            <img class="crystal-images" id="crystal3" src="assets/images/Thunder_Badge.png" alt="crystal-3">
+            <img class="crystal-images" id="crystal4" src="assets/images/Volcano_Badge.png" alt="crystal-4">
+        </div>
+
+    </div>
+
+    <div class="row">
+        <div id="totalScoreBar">
+        <h>  Your total score is: </h>  
+        </div>
+    </div>
+    <div class="row">
+        <div id="userScoreBox">
+            <h3 id=userScore> </h3>
+        </div>
+     </div>
+
+</div>
+<script type="text/javascript">
+
```

Comment:
Since you moved your javascript to another page you can just delete the commented out section here